src/ontology/edits/oeo-shared.omn

```diff
@@ -1913,6 +1913,20 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1034",
         <http://purl.obolibrary.org/obo/IAO_0000136> some OEO_00140081
     
     
+Class: OEO_00020166
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-model"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A methodology is a plan specification that describes processes that are part of a study.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/971
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1011",
+        rdfs:label "methodology"
```

Comment:
Perhaps add a term tracker as well?
```suggestion
pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1011

move to oeo-shared:
issue: https://github.com/OpenEnergyPlatform/ontology/issues/1129
pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1306"@en,
        rdfs:label "methodology"
```

Reply:
In my view, in that case, the term tracker is not mandatory as the class itself is not changed, only the file where it is stored. But it does not hurt either, so I accept this suggested change.